db/schema.rb

```diff
@@ -150,12 +156,8 @@
     t.text     "description"
     t.integer  "department_id"
     t.integer  "semester_id"
-    t.datetime "created_at",         null: false
-    t.datetime "updated_at",         null: false
-    t.string   "image_file_name"
```

Comment:
i have checked the local master . files are there, i merged master to local join table but it is not updating  ---.now let me try git merge